File: BlackBerry/AlertMsgs/ReadClipboardCmd.java
package com.mlhsoftware.Blurts;

import org.json.me.*;


class ReadClipboardCmd extends CmdBase 
{
  // Tag Names
  private static final String KEY_TEXT = "Text";


  public ReadClipboardCmd( String string ) throws JSONException
  {
    super( string );
  }

  public ReadClipboardCmd()
  {
    super( CMD_READCLIPBOARD );
  }
}

File: Desktop/Domain/CmdMsgs/ReadClipboardCmd.cs
using System;

using System.Xml;
using System.Xml.Xsl;
using System.Collections;
using System.Runtime.InteropServices;

namespace Blurts
{
  [ComVisible(true)]
  public class ReadClipboardCmd : CmdBase
  {
    // Tag Names
    //private static String KEY_TEXT = "Text";


    public ReadClipboardCmd(String s)
      : base(s)
    {
    }

    public ReadClipboardCmd()
      : base(CMD_READCLIPBOARD)
    {
    }

  }
}
